sympy/printing/latex.py

```diff
@@ -2359,6 +2363,21 @@ def latex(expr, **settings):
     $\left [ 2 / x, \quad y\right ]$
 
     """
+    settings = {
+        'fold_frac_powers' : fold_frac_powers,
+        'fold_func_brackets' : fold_func_brackets,
+        'fold_short_frac' : fold_short_frac,
+        'inv_trig_style' : inv_trig_style,
+        'itex' : itex,
+        'ln_notation' : ln_notation,
+        'long_frac_ratio' : long_frac_ratio,
+        'mat_delim' : mat_delim,
+        'mat_str' : mat_str,
+        'mode' : mode,
+        'mul_symbol' : mul_symbol,
+        'order' : order,
+        'symbol_names' : symbol_names,
+    }
 
     return LatexPrinter(settings).doprint(expr)
```

Comment:
I think changes will be required in `LatexPrinter` [class](https://github.com/sympy/sympy/blob/master/sympy/printing/latex.py#L140) as well. (CC @moorepants)

Reply:
Yes I was thinking the same, but then we need to change the Printer class as well and I was not sure whether all these changes come under the scope of the issue